netci.groovy

```diff
@@ -27,6 +27,18 @@ def prCCJob = job(Utilities.getFullJobName(project, 'code_coverage_windows', tru
   }
 }
 
+[true, false].each { isPR -> 
+  def codeFormatterJobName = Utilities.getFullJobName(project, 'native_code_format_check', isPR)
+  def codeFormatterJob = job(codeFormatterJobName) {
+    label('ubuntu')
+    seteps {
```

Comment:
seteps -> steps.  Comment 'test ci please' when fixed to rerun the CI definition generation test